radicle-cli/src/commands/patch/common.rs

```diff
@@ -33,68 +31,35 @@ pub fn branch_oid(branch: &git::raw::Branch) -> anyhow::Result<git::Oid> {
     Ok(oid.into())
 }
 
-/// List of merge targets.
-#[derive(Debug, Default)]
-pub struct MergeTargets {
-    /// Merge targets that have already merged the patch.
-    pub merged: Vec<Remote>,
-    /// Merge targets that haven't merged the patch.
-    pub not_merged: Vec<(Remote, git::Oid)>,
-}
-
-/// Find potential merge targets for the given head.
-pub fn find_merge_targets(
-    head: &Oid,
-    branch: &git::RefStr,
-    storage: &Repository,
-) -> anyhow::Result<MergeTargets> {
-    let mut targets = MergeTargets::default();
-
-    for remote in storage.remotes()? {
-        let (_, remote) = remote?;
-        let Some(target_oid) = remote.refs.head(branch) else {
-            continue;
-        };
-
-        if is_merged(storage.raw(), target_oid.into(), *head)? {
-            targets.merged.push(remote);
-        } else {
-            targets.not_merged.push((remote, target_oid));
-        }
+#[inline]
+fn get_branch(git_ref: git::Qualified) -> String {
+    let (_, _, head, tail) = git_ref.non_empty_components();
+    if tail.clone().count() == 0 {
+        return head.to_string();
     }
-    Ok(targets)
+    head.join(tail.collect::<git::RefString>()).to_string()
 }
```

Comment:
This is kind of crazy, there must be a simpler way @FintanH ? Maybe via `RefStr::strip_prefix` and `lit`?